olive/data_container/registry.py

```diff
@@ -0,0 +1,266 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import logging
+from typing import Union
+
+from olive.data_container.constants import (
+    DataComponentType,
+    DataContainerType,
+    DefaultDataComponent,
+    DefaultDataContainer,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class Registry:
```

Comment:
>Registry [](http://example.com/codeflow?start=6&length=8)

should [https://github.com/intel/neural-compressor/blob/master/neural_compressor/strategy/strategy.py#L52](https://github.com/intel/neural-compressor/blob/master/neural_compressor/strategy/strategy.py#L52) be more clear?

Reply:
Thanks for providing the registry from another view. Current approach made the similar results via different registry function.